### DISCUSSION TITLE
Filament Sensor Not working

### DISCUSSION THREAD

**Discussion by Skytrius** [bassamanator/Sovol-SV06-firmware#154] [Q&A]
Hello, I adapted the config as described, unfortunately the filament sensor doesn't work, I can do what I want 

SOVOL Sv06 
BTT Mini V3
Klipper all Updates

**Comment by bassamanator** [comment]
So, you've read the relevant section in `printer.cfg` and adjusted it?

**Reply by Skytrius** [reply]
Yes, I have, unfortunately nothing is happening

**Comment by bassamanator** [comment]
I was going to point you to the filament sensor section of the everything sv06 repo, but first, how do you even know that 'nothing is happening'?

Helpers: [Use Codeblocks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) | [Explain the shell](https://explainshell.com/) | [OSS Klipper Config](https://github.com/bassamanator/Sovol-SV06-firmware) | [Everything-Sovol-SV06](https://github.com/bassamanator/everything-sovol-sv06) | [Use for pastebin.com for large files](https://pastebin.com/)

**Reply by Skytrius** [reply]
I'll try again and let you know
